src/config/kernel.md

```diff
@@ -51,6 +51,55 @@ that is currently booted:
 # vkpurge rm all
 ```
 
+## Microcode
+
+Microcode is loaded onto the CPU or GPU at boot, by the BIOS, but can be
+replaced later in the boot stage by the OS itself. An update to microcode can
+allow a CPUs or GPUs low level behavior to be modified to work around certain
+yet to be discovered bugs, without needing to replace the hardware.
+
+### Installation
+
+#### Intel
+
+The Intel microcode package is in the non-free repo, so that has to be enabled
+first.
+
+```
+# xbps-install -S void-repo-nonfree
+```
+
+Install the `intel-ucode` package
+
+```
+# xbps-install -S intel-ucode
+```
+
+Then reconfigure the linux kernel. This only has to be done once, the first time
+the `intel-ucode` package is installed.
+
+```
+# xbps-reconfigure -f linux-X.X.X_X
+```
+
+#### AMD
+
+The AMD package contains microcode for both AMD CPUs and GPUs. AMD CPUs and GPUs
+will automatically load the microcode, no further configuration required.
+
+```
+# xbps-install -S linux-firmware-amd-<date>
```

Comment:
the package is named `linux-firmware-amd` (subpackage of [linux-firmware](https://github.com/void-linux/void-packages/tree/aa0d17eab9b00bf193d2bcf6ff6475b7ad731c78/srcpkgs/linux-firmware/template)), no need to append the version.